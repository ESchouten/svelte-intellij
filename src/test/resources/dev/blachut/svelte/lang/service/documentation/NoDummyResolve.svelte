<script lang="ts">
  let unrelatedData = {
    unrelatedValue: 42
  };

  export let usedData: any;

  console.log(unrelatedData);
</script>

{usedData.<caret>unrelatedValue}
