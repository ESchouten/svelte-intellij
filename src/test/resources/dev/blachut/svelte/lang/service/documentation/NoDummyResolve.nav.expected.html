any<br/>[ide]<br/>(property) unrelatedData.unrelatedValue: 42